src/Our.Umbraco.Ditto/Extensions/PublishedContentExtensions.cs

```diff
@@ -486,150 +512,151 @@ public static class PublishedContentExtensions
+            //                                 : propertyType.GetCustomAttribute<TypeConverterAttribute>(true));
+
+            //if (converterAttribute != null && converterAttribute.ConverterTypeName != null)
+            //{
+            //    // Time custom conversions.
+            //    using (DittoDisposableTimer.DebugDuration<object>(string.Format("Custom TypeConverter ({0}, {1})", content.Id, propertyInfo.Name)))
+            //    {
+            //        // Get the custom converter from the attribute and attempt to convert.
+            //        var converterType = Type.GetType(converterAttribute.ConverterTypeName);
+            //        if (converterType != null)
+            //        {
+            //            var converter = converterType.GetDependencyResolvedInstance() as TypeConverter;
+
+            //            if (converter != null)
+            //            {
+            //                // Create context to pass to converter implementations.
+            //                // This contains the IPublishedContent and the currently converting property descriptor.
+            //                var descriptor = TypeDescriptor.GetProperties(instance)[propertyInfo.Name];
+            //                var context = new DittoTypeConverterContext
+            //                {
+            //                    Instance = content,
+            //                    PropertyDescriptor = descriptor
+            //                };
+
+            //                Type propertyValueType = null;
+            //                if (propertyValue != null)
+            //                {
+            //                    propertyValueType = propertyValue.GetType();
+            //                }
+
+            //                // We're deliberately passing null.
+            //                // ReSharper disable once AssignNullToNotNullAttribute
+            //                if (converter.CanConvertFrom(context, propertyValueType))
+            //                {
+            //                    object converted = converter.ConvertFrom(context, culture, propertyValue);
+
+            //                    if (converted != null)
+            //                    {
+            //                        // Handle Typeconverters returning single objects when we want an IEnumerable.
+            //                        // Use case: Someone selects a folder of images rather than a single image with the media picker.
+            //                        var convertedType = converted.GetType();
+
+            //                        if (propertyIsEnumerableType)
+            //                        {
+            //                            var parameterType = typeInfo.GenericTypeArguments.First();
+
+            //                            // Some converters return an IEnumerable so we check again.
+            //                            if (!convertedType.IsEnumerableType())
```

Comment:
Have we lost the ability to return single or multiple instances using the same converter? Before I could use the picker converter to return a single item or vice versa.